docs/docs/examples/logging/efk/EFK.md

```diff
@@ -238,13 +252,13 @@ $ curl -sS -u "${ES_DOMAIN_USER}:${ES_DOMAIN_PASSWORD}" \
 ]
 '
 ```
-Finally, it is time to deploy Fluent Bit DaemonSet:
+Finally, it is time to deploy the `Fluentbit` DaemonSet:
 ```shell
-$ kubectl apply -f src/main/logging/fluentbit.yaml
+kubectl apply -f src/main/logging/fluentbit.yaml
```

Comment:
@nghazali where does the fluentbit.yaml config file come from in this case?

Reply:
That's right. This file is moved to this path `docs/docs/examples/logging/efk/managed_es/`

Reply:
Thanks